README.md

```diff
@@ -1,2 +1,158 @@
+    - [Usage ](#usage-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 ruby-group-capstone-project <a name="ruby-group-capstone-project"></a>
+In this project, we created a console app that will help you to keep a record of different types of things you own:
+
+- books
+- music albums
+- movies
+- games
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul> 
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add new book.
+- Add new games.
+- Add new music albums.
+- List all items added.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Video Link <a name="live-demo"></a>
+
+[Link](https://drive.google.com/file/d/1H18hCwa2-Xa00IpIVj-eSbV87Ze0vvB6/view?usp=sharing)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby](https://www.ruby-lang.org/en/) installed and running.
+- You should be able to use [IRB](https://en.wikipedia.org/wiki/Ruby_(programming_language)#Features).
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone git@github.com:gealsanchez/ruby-capstone.git
+```
+
+### Install
+
+Install thr project dependencies with:
+
+```
+  bundle install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  ruby main.rb
+```
+
+### Test <a name="test"></a>
+
+To test the project, execute the following command:
+
+```
+  rspec src/spec/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Fithamlak**
+
+- GitHub: [@fithamlak](https://github.com/fithamlak)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/fithamlak-fikrie)
+
+👤 **Gerson SANCHEZ**
+
+- GitHub: [@githubhandle](https://github.com/gealsanchez)
+- Twitter: [@twitterhandle](https://twitter.com/gealsanchez)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gerson-sanchez-88309b57/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+ - Implement:
+
+  - Add movies
+  - List movies
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
```

Comment:
- [ ] [OPTIONAL] I noticed that a header section like Acknowledgment is missing. Please make sure to add all the header sections are present in your as required for readme. 👍